### PR TITLE
RHPAM-1048: Stunner - Intermediate Throwing Event can have more than one outgoing flow

### PR DESCRIPTION
**Pull request by Josephblt**
https://issues.jboss.org/browse/RHPAM-1048

New SequenceFlow Rules added for max and min outgoing flows.

### CODE REVIEW

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/SequenceFlow.java [hunk 1]
```diff
@@ -64,6 +64,10 @@
 // A single outgoing sequence flows for event types that can be docked (boundary) such as Intermediate Timer Event
 @EdgeOccurrences(role = "IntermediateEventOnActivityBoundary", type = EdgeOccurrences.EdgeType.OUTGOING, max = 1)
 // Sequence flows cannot exceed bounds when any of the nodes are in an embedded subprocess context.
+
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/SequenceFlow.java]: no need for empty lines, right? :)

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
thanks @Josephblt , please remember to cherry-pick this to master.

**Pull request merged**